.github/workflows/lint.yml

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+name: Lint all codez
+on:
+  push:
+    tags: 'release-*'
+    branches:
+      - dev
+      - release
+  pull_request:
+    branches:
+      - main
+      - dev
+
+jobs:
+  go:
+    name: Lint go files
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.13.6
+      id: go
+
+    - name: install crlfmt
+      run: |
+        # Workaround to download the specific version we use internally
+        cat > go.mod << EOF
+        module vectorized
+        require (
+          github.com/cockroachdb/crlfmt v0.0.0-20200309141329-97b747f03b21
+        )
+        go 1.13
+
+        EOF
+
+        go mod download
+        go install github.com/cockroachdb/crlfmt
+        rm -rf go.mod
+
+    - name: lint go files
+      run: |
+        find . -name *.go -type f | xargs -n1 crlfmt -wrap=80 -w
+        git diff --exit-code
+
+  js:
+    name: Lint js files
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up Node
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: '12.16.1'
+
+    - name: Set up prettier
+      run: npm install prettier
+
+    - name: Lint JS files
+      run: npx prettier --list-different src/**/*.{ts,js}
+
+  py:
+    name: Lint python files
+    runs-on: ubuntu-20.04
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up python
+      run: |
+        sudo apt-get install -y python3 python3-pip
+        sudo pip install yapf
+
+    - name: lint python files
+      run: find . -name *.py -type f | xargs -n1 yapf -d
+
+  cpp:
+    name: Lint files with clang-format
+    runs-on: ubuntu-latest
+    # Use purpose built container since the latest llvm versions we use are
+    # not always available as packages. This container can also get used by contributors locally.
+    container: 0xdiba/clang-format:latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Run clang fmt
+      run: |
+        find . -regex '.*\.\(cpp\|h\|hpp\|cc\|proto\|java\)' | xargs -n1 clang-format -i -style=file -fallback-style=none
+        git diff --exit-code
+
+  sh:
+    name: Lint shell scripts
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: setup shfmt
+      run: |
+        wget "https://github.com/mvdan/sh/releases/download/v3.2.0/shfmt_v3.2.0_linux_amd64" -O shfmt
+        chmod +x shfmt
+
+    - name: Run shfmt
+      run: ./shfmt -i 2 -ci -s -d .
```

Comment:
Missing new line

Reply:
still missing a new line. if you're using vscode, there is a setting that will do this automatically.